src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py

```diff
@@ -189,6 +189,11 @@
     short-summary: get publishing endpoints, credentials, database connection strings, etc
 """
 
+helps['webapp deployment list-container-cd-url'] = """
+    type: command
+    short-summary: get webhook url for your web app on Linux to configure your container repo for continuous deployment setup
```

Comment:
Suggest: ` get the webhook url of the linux webapp you can use to configure the container repository for continuous deployment`